test/integration/targets/lookup_mongodb/tasks/test_lookup_mongo.yml

```diff
@@ -0,0 +1,62 @@
+- name: "assert we complain properly if pymongo is not there"
+  debug: msg="{{ lookup('mongodb', mongodb_parameters ) }}"
+  register: result
+  ignore_errors: yes
```

Comment:
The test should not assume that `pymongo` is not already installed.

Reply:
This doesn't look like it's been resolved

Reply:
I believe it is. I check if "pymongo is required" exactly 5 lines below this one.

Reply:
I think the problem is that `pymongo` may already be installed